flocker/common/script.py

```diff
@@ -164,15 +164,15 @@ def eliot_output(message):
         action_type = message.get('action_type')
         action_status = message.get('action_status')
 
-        message_format = '%s'
+        message_format = ''
```

Comment:
@james-w The messages still go to other destinations if configured. E.g. `run-acceptance-tests` and `setup-cluster` add `FileDestination` so that all eliot log messages go to a log file.